tor_core/helpers.py

```diff
@@ -79,23 +79,23 @@ def clean_list(items):
     return cleaned
 
 
-def send_to_slack(message, config):
+def send_to_slack(message, config,  channel="#general"):
     """
-    Sends a message to the ToR #general slack channel.
+    Sends a message to the ToR slack.
 
-    :param message: String; the message that is to be encoded.
+    :param message: String; the message that is to be encoded
+    :param channel: channel option, defaults to general
```

Comment:
Let's add in that this should be a string as well -- we're not going full-on type hinting, but I like to have it in the docstrings.